taxi/views.py

```diff
@@ -64,13 +73,13 @@
 
 class CarCreateView(LoginRequiredMixin, generic.CreateView):
     model = Car
-    fields = "__all__"
+    form_class = CarCreationForm
     success_url = reverse_lazy("taxi:car-list")
 
 
 class CarUpdateView(LoginRequiredMixin, generic.UpdateView):
     model = Car
-    fields = "__all__"
+    form_class = CarCreationForm
```

Comment:
The `CarUpdateView` also uses `CarCreationForm`. If the update form requires different fields or validations, consider creating a separate form class for updates.